### DISCUSSION TITLE
is it possible to update a cron workflow in argo via api ?

### DISCUSSION THREAD

**Discussion by ghost** [argoproj/argo-workflows#8606] [Q&A] [answered]
I tried to update a workflow created via API by first fetching the current cron workflow and sending a PUT request using the response obtained via GET call(after only changing the workflow spec), is this the approach supported, where did i go wrong?
The following is the response 
```{
    "code": 13,
    "message": "runtime error: invalid memory address or nil pointer dereference"
}

**Comment by terrytangyuan** [comment, marked as answer]
Make sure the spec after you modified is still valid?

**Reply by ghost** [reply]
hi @terrytangyuan, Yes it was a spec problem the JSON format where misformed !! 
thanks for the help

**Comment by alexec** [comment]
You should never update cron workflow. It can result in the workflow not being scheduled.

**Reply by ghost** [reply]
@alexec could you please help me understand the scenario where the workflow not being scheduled?

**Reply by alexec** [reply]
If the cron is changed is is removed and re-added to the scheduler. It if would have scheduled when that happen, it will not run.

**Reply by ghost** [reply]
oh my!!, thanks for the info, I will keep that in  mind

**Comment by weironz** [comment]
v3.6.0 support `argo cron update`
